### PR TITLE
Substitute /dev/ instead of translating it

### PR DESCRIPTION
**Pull request by nmollerup**
## Pull Request Checklist

**Is this in reference to an existing issue?**
https://github.com/sensu-plugins/sensu-plugins-disk-checks/issues/105
#### General

- [x] Update Changelog following the conventions laid out [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)

- [x] Update README with any necessary configuration snippets

- [x] Binstubs are created if needed

- [x] RuboCop passes

- [x] Existing tests pass

#### New Plugins

- [x] Tests

- [x] Add the plugin to the README

- [x] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

### CODE REVIEW

**Comment by majormoses** [comment]
Thanks for your contribution to Sensu plugins! Without people like you submitting PRs we couldn't run the project. I will review it shortly.

Apologies on the long delay for review.

File: bin/metrics-disk-capacity.rb [hunk 1]
```diff
@@ -68,7 +68,7 @@ def run
 
         timestamp = Time.now.to_i
         if fs =~ /\/dev/
-          fs = fs.tr('/dev/', '')
+          fs = fs.sub('/dev/', '')
```
Comment (majormoses) [inline bin/metrics-disk-capacity.rb]: I had not used `sub` (very familiar with `gsub`) before and had to compare them from what I can tell the difference is that  it replaces only the first occurrence as opposed to all occurrences with `gsub` and `tr`. Is that right?

Comment (majormoses) [reply bin/metrics-disk-capacity.rb]: I have updated the changelog to reflect this as I think it's pretty nuanced and not obvious without inspecting the documentation for each side by side.

**Comment by majormoses** [comment]
I have cut a release but our CI (travis is backed up) I will check back in a few hours and if its not released I will manually publish to rubygems.

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-disk-checks/versions/3.1.1

**Comment by majormoses** [comment]
Great work

File: bin/metrics-disk-capacity.rb (hunk 1 above)
Comment (nmollerup) [reply bin/metrics-disk-capacity.rb]: `tr` translates every occurrence of the characters, `sub` substitutes the first occurrence of the specified string, whereas `gsub` substitutes all occurrences of the string.

**Pull request merged**